core/src/cg/group4/game_logic/stroll/events/TestStrollEvent.java

```diff
@@ -112,57 +90,20 @@ public void update(final Observable o, final Object arg) {
         cDelayInputTimer = new Timer("DELAYEVENTINPUT", 1);
         cDelayInputTimer.getStartSubject().addObserver(cDelayInputStartObserver);
         cDelayInputTimer.getStopSubject().addObserver(cDelayInputStopObserver);
-        TimerStore.getInstance().addTimer(cDelayInputTimer);
 
         cDelayNewInput = false;
 
         cAccelMeter = new Accelerometer(StandUp.getInstance().getSensorReader());
-        cAccelMeter.filterGravity(true);
-        
-        doTask();
     }
 
     /**
      * Sets the new operation that should be done.
      */
     public final void doTask() {
```

Comment:
Yeah it is a weird name however this was already named this way.. The event itself should also be renamed. However this can be done in a later state.. don't feel like waiting for Travis CI again...